assignment1.txt

```diff
@@ -0,0 +1,30 @@
+\*Rebecca Hale, Part 2 Assignment 1*\
+
+#include <stdio.h>
+  int main(){
+
+	\*Question 1*\
+  	printf("A compiler is a tool used by computers that translates/n");
+	printf("code written by humans into code/n");
+	printf("that computers can understand or follow.\n");
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.